online_meetings/canvas_zoom_meetings.py

```diff
@@ -52,6 +52,9 @@ def zoom_course_report(canvas_account=1, enrollment_term_id=1, published=True):
     per_page = 100
     # Get all published courses from the defined enrollment term
     courses = account.get_courses(enrollment_term_id=enrollment_term_id, published=published, per_page=per_page)
+    # For testing
+    # course = CANVAS.get_course(331376)
+    # courses = [course, ]
```

Comment:
Do we need this?

Reply:
No, I was thinking about removing it, but when I need to test I forget what the call is. Maybe I need to just make it more configurable really, either run on a term or on a specific course.

Reply:
Perhaps, yeah. I like including a test mode in some of my projects, which just slices lists down to a reasonable number before processing. I haven't gotten around to implementing that here yet.

Reply:
I'll create a new issue that will reference removing this and supplying a term or course as part of a refactor. It's on #53. We'll do this after.